test/integration/job/job_test.go

```diff
@@ -286,6 +286,154 @@ func TestJobFinishedNumReasonMetric(t *testing.T) {
 	}
 }
 
+func TestJobFinishedIndexesTotalMetric(t *testing.T) {
```

Comment:
I've added the tests as integration, because this is what we did so far for most features, however, with the recent issues with test timeouts for integration I'm thinking about moving them to unit tests. WDYT @alculquicondor ?

Reply:
Can you just add checks to the existing tests?

Reply:
I think I can, let me try